tools/release.sh

```diff
@@ -52,7 +52,7 @@ function strip-and-split-debuginfo {
 }
 
 function get-firecracker-version {
-    (cd src/firecracker; cargo pkgid | cut -d# -f2 | cut -d: -f2)
+    (cd src/firecracker; cargo pkgid | cut -d# -f2 | (echo -n v; cut -d: -f2))
```

Comment:
nit: how about having it in the head?
```suggestion
    (cd src/firecracker; echo -n v; cargo pkgid | cut -d# -f2 | cut -d: -f2)
```

Reply:
That looks better. We should forward-port this version to `main`